src/effects/DynamicRangeProcessorEditor.cpp

```diff
@@ -19,7 +19,7 @@
 #include <wx/textctrl.h>
 
 #if wxUSE_ACCESSIBILITY
-#include "WindowAccessible.h"
+#   include "WindowAccessible.h"
```

Comment:
unintentional change?

Reply:
Same reason as above, but that one looks peculiar all right ...